### DISCUSSION TITLE
Markdown

### DISCUSSION THREAD

**Discussion by sfnagle** [usnistgov/NEMO#257] [Ideas]
Is there any interest in implementing markdown for tool description text instead of only html support? I could not find previous discussion of this.

**Comment by rptmat57** [comment]
I never really thought about this but that's an interesting idea.

Technically speaking, to implement this we would probably need a new dropdown in detailed admin to pick the format (unless we can support both directly) and we would need to add a new dependency on the python-markdown library.
In general I am not a fan of adding new dependencies, but this could potentially be useful in other areas of NEMO like news etc.

If someone wants to dig deeper and work on this I am all ears
